networking/multiple_networks/configuring-multi-network-policy.adoc

```diff
@@ -8,13 +8,6 @@ toc::[]
 
 As a cluster administrator, you can configure multi-network for additional networks. You can specify multi-network policy for SR-IOV, macvlan, and OVN-Kubernetes additional networks. Macvlan additional networks are fully supported. Other types of additional networks, such as ipvlan, are not supported.
 
-[IMPORTANT]
-====
-Support for configuring multi-network policies for SR-IOV additional networks is a Technology Preview feature and is only supported with kernel network interface cards (NICs). SR-IOV is not supported for Data Plane Development Kit (DPDK) applications.
```

Comment:
mutli-network policies for SR-IOV additional network is still only supported with kernel network interface, even if it turned GA.
I'd still keep 

```
Support for configuring multi-network policies for SR-IOV additional networks is only supported with kernel network interface cards (NICs). SR-IOV is not supported for Data Plane Development Kit (DPDK) applications.
```